modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/SearchTemplateResponse.java

```diff
@@ -48,6 +48,12 @@
     SearchTemplateResponse() {
     }
 
+    SearchTemplateResponse(StreamInput in) throws IOException {
```

Comment:
I wonder if we can remove an empty constructor and make all class variables final here? We may need to change a little `fromXContent` and have another  constructor.

Reply:
this is possible yet it requires some work on how SearchTemplateResponse is used, as source and response are generally set after creation. I would prefer to keep this out of this PR.